docs/input/react.md

```diff
@@ -0,0 +1,155 @@
+## Import
+
+```js
+import { TextField } from '@warp-ds/react';
+```
+
+## Usage
+
+```js
+<TextField label="Name" />
+```
+
+## Migrating from Troika
```

Comment:
I don't think we need to include Migrating from Troika sections in Warp docs. Applications should've already been migrated to Fabric by the time Warp is introduced, right @Skadefryd?

Reply:
Fixed here https://github.com/warp-ds/tech-docs/pull/10/commits/b505a31bf7853cc00189c8896cb030a8d99ba74a